### DISCUSSION TITLE
SMTP relays feature not working

### DISCUSSION THREAD

**Discussion by rccess** [postalserver/postal#1456] [Help with using Postal]
i have configured 

SMTP relays
----------------------------
smtp_relays:
   -
    hostname: lax-tls.heficed.com
    port: 587
    ssl_mode: Auto
---------------------------
![image](https://user-images.githubusercontent.com/32220165/125310898-9f092e80-e350-11eb-8154-dac2d5bfd209.png)

issue with soft fail

"mail destination not given"

![image](https://user-images.githubusercontent.com/32220165/125311008-b3e5c200-e350-11eb-9177-cae231e04f3c.png)

**Comment by willpower232** [comment]
It sounds like that is a response from the proxy, are you hosting the proxy yourself or can you ask whoever does run the proxy about that message?

**Comment by modestas-j** [comment]
@willpower232 any chance you know how to configure TRANSPORT MAP in postal ? Similarly in postfix it's configured by adding "transport_maps = hash:/etc/postfix/transport" in main.cf and then adding the following in the transport file:

example-domain.com smtp: lax-tls.heficed.com:587 
postmap hash:/etc/postfix/transport

How to achieve this in postal ?

**Comment by willpower232** [comment]
I don't think Postal has such a feature, it is currently set up to send all email to the relay I believe.
